Marlin/src/gcode/calibrate/G34_M422.cpp

```diff
@@ -110,11 +110,15 @@ void GcodeSuite::G34() {
       break;
     }
 
-    const float z_auto_align_amplification = parser.floatval('A', Z_STEPPER_ALIGN_AMP);
-    if (!WITHIN(ABS(z_auto_align_amplification), 0.5f, 2.0f)) {
-      SERIAL_ECHOLNPGM("?(A)mplification out of bounds (0.5-2.0).");
-      break;
-    }
+    const float z_auto_align_amplification =
+      #if ENABLED(Z_STEPPER_ALIGN_KNOWN_STEPPER_POSITIONS)
+        Z_STEPPER_ALIGN_AMP;
+      #else
+        parser.floatval('A', Z_STEPPER_ALIGN_AMP);
+        if (!WITHIN(ABS(z_auto_align_amplification), 0.5f, 2.0f)) {
```

Comment:
Close } appears to be missing here

Reply:
Oops, missed a commit when I pushed those changes.